db/schema.rb

```diff
@@ -89,7 +89,7 @@
     t.index ["name"], name: "index_currencies_on_name", unique: true
   end
 
-  create_table "recurring_payments", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "recurring_payments", options: "ENGINE=InnoDB DEFAULT CHARSET=latin1", force: :cascade do |t|
```

Comment:
Could you please rollback that line change? I think that's something specific to your local MySQL setup.

Reply:
```suggestion
  create_table "recurring_payments", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```